src/integrationTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerEndpointTest.java

```diff
@@ -324,23 +312,8 @@ void dlq_case_event_message_should_be_stored_and_return_200_ok() throws Exceptio
             assertEquals(0, response.getRetryCount(), "Valid RetryCount should be returned");
         }
 
-        @Test
-        void dlq_case_event_message_should_not_be_stored_and_return_200_ok_when_feature_flag_disabled()
```

Comment:
Should we be removing this test when the assertion is to do a particular thing based on the feature flag? I.e. removing the flag should be sufficient and the test should execute with the expected R2 journey.

Reply:
We have removed the logic of R1 route code as part of this changes, so R1 tests (when_feature_flag_disabled) have to be removed too.